src/dmd/frontend.h

```diff
@@ -1249,6 +1249,7 @@ class Type : public ASTNode
     static ClassDeclaration* typeinfoshared;
     static ClassDeclaration* typeinfowild;
     static TemplateDeclaration* rtinfo;
+    static TemplateDeclaration* rttypeid;
```

Comment:
So basically until we can switch fully to the auto-generated headers, updates to the frontend header file are a strong indication that the old header files need to be manually updated as well.